src/picker/selector/choose.py

```diff
@@ -0,0 +1,58 @@
+from subprocess import run
+from typing import Dict, List, Tuple, Union
+
+from ..abstractionhelper import is_installed, is_macos
+from ..models import CANCEL, DEFAULT, Action, CharacterEntry, Shortcut
+from .selector import Selector
+
+
+class Choose(Selector):
+    @staticmethod
+    def supported() -> bool:
+        return is_installed("choose") and is_macos()
```

Comment:
1) Here, the order is different: Only check for `is_macos` if `is_installed`. I'd prefer the same order in all helpers; and maybe `is_macos` first is more intuitive? What do you think?

2) Again, do we even need `is_macos`?